test/contacts.js

```diff
@@ -1,14 +1,35 @@
-const chai = require('chai')
-const expect = chai.expect
+const { expect } = require('chai')
+const fakeHubspotApi = require('./helpers/fake_hubspot_api')
 
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
 const _ = require('lodash')
+const emailsFromContacts = contacts =>
+  contacts.flatMap(contact =>
+    contact['identity-profiles'].map(
+      profile =>
+        profile.identities.find(identity => identity.type === 'EMAIL').value,
+    ),
+  )
 
 describe('contacts', function() {
-  this.timeout(10000)
+  const hubspot = new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
 
   describe('get', function() {
+    const contactsGetEndpoint = {
+      path: '/contacts/v1/lists/all/contacts/all',
+      response: { contacts: [{}] },
+    }
+    const tenContactsGetEndpoint = {
+      path: '/contacts/v1/lists/all/contacts/all',
+      response: { contacts: [{}, {}, {}, {}, {}, {}, {}, {}, {}, {}] },
+      query: { count: 10 },
```

Comment:
Does this work similar to an expectation where calling `hubspot.contacts.get({ count: 5 })` would cause this test to fail because the count didn't match the mocked endpoint? 

If that's true, what do you think of removing the assertion: `expect(data.contacts.length).to.be.at.most(10)`? 

I believe this expectation is based on the set response, which isn't very helpful or informative (IMO). I think the real value is asserting the correct endpoint is called, with the correct params, and the response matches an expected object (in this case, an array). Thoughts?

Reply:
done in 
https://github.com/MadKudu/node-hubspot/pull/140/commits/c3ec3aed32e3df408aba03a6737d3f0cb740a4b3